packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -179,9 +178,9 @@ class ServiceConnectionManager {
     // race conditions where managers cannot listen for events soon enough.
     isolateManager.vmServiceOpened(service);
     consoleService.vmServiceOpened(service);
-    serviceExtensionManager.vmServiceOpened(service, connectedApp);
+    serviceExtensionManager!.vmServiceOpened(service, connectedApp);
```

Comment:
make serviceExtensionManager.vmServiceOpened accept a non-null connectedApp. Then unwrap it here in the method call to be consistent with timelineStreamManager.vmServiceOpened. Also, we should never be calling vmServiceOpened with a null connectedApp, so no need to accept a nullable type

Reply:
done. 
Noted serviceExtensionManager.vmServiceOpened  should not accept null. It is not null safe yet.